ProductsWebApi/ProductsWebApi/Services/AuthenticationServices/UserService.cs

```diff
@@ -0,0 +1,36 @@
+﻿using FluentValidation;
+using Microsoft.AspNetCore.Identity;
+using ProductsWebApi.Interface;
+using ProductsWebApi.Models;
+using ProductsWebApi.Validator;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ProductsWebApi.Services.AuthenticationServices
+{
+    public class UserService : IUserService
+    {
+        private readonly IUserRepository _userRepository;
+        private readonly UserValidator _userValidator;
+
+        public UserService(IUserRepository userRepository, UserValidator userValidator)
+        {
+            _userRepository = userRepository;
+            _userValidator = userValidator;
+        }
+
+        public bool IsValidUser(Users userLoginModel)
+        {
+            _userValidator.ValidateAndThrow(userLoginModel);
+
+            var user = _userRepository.FetchUserDetails(userLoginModel.UserName, userLoginModel.Password);
+            if (user == null)
+            {
+                throw new ValidationException(ValidationMessages.InvalidUserCredentials);
+            }
+            return true;
```

Comment:
What if `user` is returned as false, still in that case also you are returning hardcoded `true`?

Reply:
If user returned fall i will be throwing validation exception message